ffi/ffi_java/ffi_java_vcl/src/lib.rs

```diff
@@ -21,9 +21,11 @@ use wedpr_ffi_common::utils::{
 };
 use wedpr_s_verifiable_confidential_ledger;
 
-use wedpr_l_protos::generated::zkp::BalanceProof;
-use wedpr_s_protos::generated::vcl::{
-    BatchCreditBalanceProof, EncodedConfidentialCredit, EncodedOwnerSecret,
+use wedpr_s_protos::generated::{
+    vcl::{
+        BatchCreditBalanceProof, EncodedConfidentialCredit, EncodedOwnerSecret,
```

Comment:
add all protobuf generated with PB prefix

Reply:
This change is a bit big, if there is time, it will be fix this in the future.